scss/_patterns_menu-toggle.scss

```diff
@@ -0,0 +1,33 @@
+@mixin vf-p-menu-toggle {
+  .p-menu-toggle {
+    display: flex;
+  }
+
+  .p-menu-toggle__button {
+    // reset default button styling
+    background: none;
+    border: 0;
+    display: block;
+    line-height: inherit;
+    margin: 0;
+    margin-left: auto;
```

Comment:
why not use the align utilities instead of hardcoding algnment?

Reply:
If we want this to be a pattern it should come with some default styling.
Utilities should be used to override some styling if necessary. I don't think by default we should make use of utility classes as part of patterns.

I guess the only exception would be show/hide utilities that may be used in JavaScript to toggle visibility of component, like in this case.